ballerina-tests/http-interceptor-tests/tests/interceptors_error_handling_tests.bal

```diff
@@ -21,35 +21,31 @@ import ballerina/http_test_common as common;
 final http:Client noServiceRegisteredClientEP = check new("http://localhost:" + noServiceRegisteredTestPort.toString(), httpVersion = http:HTTP_1_1);
 
 listener http:Listener noServiceRegisteredServerEP = new(noServiceRegisteredTestPort,
-    httpVersion = http:HTTP_1_1,
-    interceptors = [
-        new LastResponseInterceptor(), new DefaultResponseErrorInterceptor(), new DefaultRequestInterceptor(),
-        new DefaultRequestErrorInterceptor(), new LastRequestInterceptor(), new DefaultResponseInterceptor()
-    ]
+    httpVersion = http:HTTP_1_1
 );
 
 @test:Config{}
 function testNoServiceRegistered() returns error? {
```

Comment:
We could remove this test right? Now it has no usecase

Reply:
Removed